### DISCUSSION TITLE
Demystifying Obsidian’s Outlining Superpowers

### DISCUSSION THREAD

**Discussion by TfTHacker** [vslinko/obsidian-outliner#428] [General]
FYI: I Covered Outliner and Zoom Plugins as a part of Obsidian's overall outlining potential. Tnx for this plugin!

https://medium.com/@tfthacker/demystifying-obsidians-outlining-superpowers-20c077793356

**Comment by vslinko** [comment]
@TfTHacker thank you for the article! btw, drag-and-drop support [was released 3 days ago](https://github.com/vslinko/obsidian-outliner/releases/tag/4.5.0)!

**Comment by TfTHacker** [comment]
Wow @vslinko this is cool. I will make a revision and also mention this on Twitter. Make sure it gets some airtime.
